tests/basic/storage_pool_test.go

```diff
@@ -6359,8 +6360,8 @@ var _ = Describe("{VerifyPoolDeleteInvalidPoolID}", func() {
 		}
 
 		commonText := "service mode delete pool.*unable to delete pool with ID.*[0-9]+.*cause.*"
-		compileText := fmt.Sprintf("[%s]operation is not supported", commonText)
-		compileTextMaintenanceError := fmt.Sprintf("[%s]Requires pool maintenance mode", commonText)
+		compileText := fmt.Sprintf("%soperation is not supported", commonText)
```

Comment:
add space after indentifier

Reply:
this is a regex expression , so ".*cause.*operation"